pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -40,21 +40,22 @@ import (
 
 	kubeversion "k8s.io/kubernetes/pkg/version"
 
+	"github.com/openshift/origin/pkg/cmd/server/crypto"
+	"github.com/openshift/origin/pkg/version"
+
+	"github.com/openshift/image-registry/pkg/clientcmd"
```

Comment:
The copy is fine, but I think you should create a package `origin-copy` or some such so that we can keep them all together and see if, when, and how we should be pulling these out into a common library.